admin/bin/dev-shell

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -eu
```

Comment:
Other shell files also do an “-o pipefail” - any reason to not have that the same?

Reply:
no reason I just forgot it, thanks for catching this !